http/src/main/java/io/micronaut/http/HttpResponse.java

```diff
@@ -68,7 +70,9 @@ default HttpResponse<B> setAttribute(CharSequence name, Object value) {
 
     /**
      * @return The HTTP status
+     * @deprecated To support custom status codes. Use {@link #code()} instead of {@link #status()} and {@link HttpStatus#getCode()}  and {@link #reason()} instead of {@link #status()} and {@link HttpStatus#getReason()}
```

Comment:
```suggestion
     * To support custom status codes. Use {@link #code()} instead of {@link #status()} and {@link HttpStatus#getCode()}  and {@link #reason()} instead of {@link #status()} and {@link HttpStatus#getReason()}
```